Modules/Devices/src/Devices.Domain/Entities/Identities/Identity.cs

```diff
@@ -13,7 +13,11 @@ public class Identity : Entity
 {
     private readonly List<IdentityDeletionProcess> _deletionProcesses;
     protected virtual EfCoreFeatureFlagSet EfCoreFeatureFlagSetDoNotUse { get; } = [];
+
+    // ReSharper disable once ReplaceWithFieldKeyword
+#pragma warning disable IDE0032
     private TierId? _tierId;
+#pragma warning restore IDE0032
```

Comment:
.NET 10 introduces the `field` key word (see https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/field) and there is a compiler warning that tells you to use it. Since we treat all compiler warnings as errors, we have to ignore the warning here. Because in this specific case it's impossible to use it (or at least I couldn't figure out a good way to do so)